detox/android/detox/src/full/java/com/wix/detox/reactnative/ReactNativeExtension.kt

```diff
@@ -62,6 +62,15 @@ object ReactNativeExtension {
         }
     }
 
+    @JvmStatic
+    fun enableAllSynchronization(applicationContext: ReactApplication) {
+        val reactContext = getCurrentReactContextSafe(applicationContext)
+
+        if (reactContext != null) {
```

Comment:
@d4vidi , this just made me wonder – do we still not support properly non-React Android apps (e.g. using network idling resources regardless of whether it is React or not)?

Reply:
Afraid so: #2543